pkg/daemon/rpm-ostree.go

```diff
@@ -105,8 +106,35 @@ func (r *RpmOstreeClient) loadStatus() (*rpmOstreeState, error) {
 	return &rosState, nil
 }
 
+// See https://bugzilla.redhat.com/show_bug.cgi?id=2111817
+func bug2111817Workaround() error {
+	targetUnit := "/run/systemd/system/rpm-ostreed.service.d/bug2111817.conf"
```

Comment:
Note that the MCD is writing the file explicitly into `/run`.  That means it's transient, and will go away on reboot (and be reapplied the next time the MCD lands - hence when we drop the workaround from the MCD, it will automatically go away).

Reply:
Ah yeah ok I see. That's fine then. I am +1 to shipping this since its only transient